src/ASM/LR/ASMu2Dnurbs.C

```diff
@@ -76,7 +78,13 @@ bool ASMu2Dnurbs::read (std::istream& is)
   } else { // probably a SplineSurface, so we'll read that and convert
     tensorspline = new Go::SplineSurface();
     is >> *tensorspline;
-    lrspline.reset(createFromTensor(tensorspline));
+    if (tensorspline->rational())
+      lrspline.reset(createFromTensor(tensorspline));
+    else {
+      lrspline.reset(new LR::LRSplineSurface(tensorspline));
+      std::cout << "LR-nurbs requested but input is a spline" << std::endl;
+      isNurbs = false;
```

Comment:
Guess we need to set the isNurbs/noNurbs flag also when reading an lr-file.
I presume that file format does not account for LR nurbs yet?